docs/SETUP.md

```diff
@@ -76,11 +76,16 @@ $ git checkout remotes/origin/master
 Note: checking out 'remotes/origin/master'.
 
 $ git pull
+```
+Eventgen uses [Poetry](https://github.com/python-poetry/poetry) to manage and build. Install `Poetry` first:
+```
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
```

Comment:
Can you also add instructions for installing poetry via pip?

Reply:
Thanks, Jack.added.